wdl-lint/tests/lints/command-mixed-line-cont/source.wdl

```diff
@@ -1,4 +1,4 @@
-#@ except: NoCurlyCommands, DescriptionMissing
+#@ except: NoCurlyCommands, DescriptionMissing, RuntimeSectionKeys
```

Comment:
From here on down, it's mostly excepting the new rule and changes to the ordering of non-related tests. You can take a look at `runtime-keys-wdl-1.0` and `runtime-keys-wdl-1.1` for the most important new tests.